Packages/io.chainsafe.web3-unity/Runtime/Scripts/CountlyAnalytics.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ChainSafe.Gaming.Web3.Analytics;
+using Plugins.CountlySDK;
+using Plugins.CountlySDK.Models;
+using UnityEngine;
+
+public class CountlyAnalytics : IAnalyticsClient
+{
+    private const string AppKey = "9fad27b7383f5151a316505ca3245287fc44ba50";
+    private const string ServerUrl = "https://trial-chainsafe.count.ly";
+
+    
+    public async Task CaptureEvent(AnalyticsEvent eventData)
+    {
+        await Countly.Instance.Events.RecordEventAsync(eventData.EventName, new Dictionary<string, object>()
+        {
+            { "chain", eventData.ChainId},
+            { "network", eventData.Network},
+            { "project-id", eventData.ProjectId},
+            { "rpc ", eventData.Rpc},
+            { "version", AnalyticsVersion},
+        } );
+    }
+
+    public string AnalyticsVersion => "2.5.5";
```

Comment:
Can we somehow read this from a global source?

Reply:
Maaaaybe some sort of a centralized utils class or something where we can store that info.